content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -291,6 +308,32 @@ Here, we first check that the optional parameter is present with `!has(params.op
   evaluated, and optionalNumber will be checked to ensure that it contains a value between 5 and
   10 inclusive.
 
+
+##### Per-Namespaced Parameters
+
+Authors of policy bindings may choose to specify cluster-wide, or per-namespace
+parameters. Normally, parameters will be searched for only in the `namespace` provided
+in the binding's `paramRef`.
+
+However, if `namesapce` is unset, the `namespace` for the request object can be
+used to search params. This enables policy configuration that depends on the namespace
+of the resource being manipulated, for more fine-tuned control.
+
+##### Parameter Selector
+
+In addition to specify a parameter in a binding by `name`, you may
+choose instead to specify a matching label, such that all resources of the
```

Comment:
Is it just one label or can use use any non-field selector?

You can write `{{< glossary_tooltip text="selector" term_id="selector">}}` to link to the docs for selector.